manifests/g/Google/JapaneseIME/2.28.5050.0/Google.JapaneseIME.installer.yaml

```diff
@@ -0,0 +1,14 @@
+# Created with WinGet Automation using Komac v1.10.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Google.JapaneseIME
+PackageVersion: 2.28.5050.0
+InstallerType: msi
+ProductCode: '{B687E56C-1666-41E9-9D34-7810E0A5A6F5}'
+ReleaseDate: 2023-05-11
+Installers:
+- Architecture: x64
+  InstallerUrl: https://edgedl.me.gvt1.com/edgedl/release2/acoyxcj7sytib4tlms42pohlehyq_2.28.5050.0/GoogleJapaneseInput64-2.28.5050.0.msi

```

Comment:
I can't verify this InstallerUrl - the PackageUrl gives me a user-specific URL such as `https://dl.google.com/tag/s/appguid%3D%7BDDCCD2A9-025E-4142-BCEB-F467B88CF830%7D%26iid%3D%7BC46E951B-B03B-7CDE-9F61-BC4C2FB8E47B%7D%26lang%3Den%26browser%3D4%26usagestats%3D0%26appname%3DGoogle%2520Japanese%2520Input%26needsadmin%3Dtrue%26ap%3Dexternal-stable-universal/japanese-ime/GoogleJapaneseInputSetup.exe`.